apparmor.d/profiles-a-f/cc-remote-login-helper

```diff
@@ -0,0 +1,17 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{libexec}/cc-remote-login-helper
+profile cc-remote-login-helper @{exec_path} flags=(complain) {
```

Comment:
Do not add `complain` here. Add it in the `dists/flags/main.flag` file.

Reply:
Are any flags discouraged or just complain?

Reply:
Just complain. As it should be removed at some point,  I do it this way as it is easier to have a central place to maintain the flags for all unstable profiles.